### ISSUE TITLE
Disallow giving to own team in UI

### ISSUE THREAD

**Issue by mattbk** [liberapay/liberapay.com#767]
https://github.com/liberapay/liberapay.org/issues/12#issuecomment-340067838

>although it looks technically possible to donate to a team one is on, but doesn't seem to work (I assume payday is ignoring those transfers). I know this [isn't allowed](https://liberapay.com/about/teams) ("Teams are for peer-to-peer donations, they can't be used by a legal entity to distribute donations to its members."), but the UI should prevent this in the first place (rather than causing confusion by appearing to set a donation and then it not working on payday).

**Comment by Changaco** [comment]
I don't think we want to prevent/remove a donation to a team one is a member of. However we should continue improving the system so that such a donation is clearly marked as inactive and not counted in the team's income when it's a self-donation (e.g. when the donor is also the only member of the team with a take higher than zero).

**Comment by Changaco** [comment]
> although it looks technically possible to donate to a team one is on, but doesn't seem to work (I assume payday is ignoring those transfers).

To clarify: it is possible to donate to a team one is a member of, but only if there are other people in the team who can receive the funds, since it's impossible to transfer money to oneself.

**Comment by mattbk** [comment]
Ahh, I see.

**Comment by Changaco** [comment]
This issue isn't completely fixed but it's much less likely to arise now that donations must be paid in advance to be considered active.
